src/main/java/qna/domain/Answer.java

```diff
@@ -3,15 +3,41 @@
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.ForeignKey;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
 import java.util.Objects;
 
-public class Answer {
+@Entity
+public class Answer extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    private Long writerId;
-    private Long questionId;
+
+    @Lob
     private String contents;
+
+    @Column(nullable = false)
     private boolean deleted = false;
 
+    @JoinColumn(name = "question_id", foreignKey = @ForeignKey(name = "fk_answer_to_question"))
+    @ManyToOne(fetch = FetchType.LAZY)
+    private Question question;
+
+    @JoinColumn(name = "writer_id", foreignKey = @ForeignKey(name = "fk_answer_writer"))
+    @ManyToOne(fetch = FetchType.LAZY)
+    private User writer;
```

Comment:
```suggestion
    @JoinColumn(name = "question_id", foreignKey = @ForeignKey(name = "fk_answer_to_question"))
    @ManyToOne(optional = false, fetch = FetchType.LAZY)
    private Question question;

    @JoinColumn(name = "writer_id", foreignKey = @ForeignKey(name = "fk_answer_writer"))
    @ManyToOne(optional = false, fetch = FetchType.LAZY)
    private User writer;
```
`Answer` 연관관계에서 `User` 와 `Question` 이 필수라면 optional = false 속성을 명시적으로 선언해줘도 좋을 것 같네요

Reply:
명시적으로 작성하는 것 좋은것 같아요!!